### PR TITLE
Disable Revolutionary's Pending a Rules and Mechanics Rework

### PR DESCRIPTION
**Pull request by UltimateJester**
## About the PR

Revs has been reduced to a weight of 0.00 to stop it from rolling while discussion take place on how to improve the gamemode

## Why / Balance

I have observed quite a few conversation about revs and one of the standout points that I saw was

1. "Revs doesn't work well with the rules and I feel like I'm skirting the lines of the rules when fighting revs."

2. "The mechanics feel half baked and aren't encouraging exile or even finding a way to convert the heads themselves."

3. "A way to remove mindshields should be possible alongside directing the gamemode to properly follow it's own mechanics."

The primary issues with revs is apparent with revs just turning into TDM with a label, there is not attempt at exiling nor is there any fun in security being extremely paranoid and shooting anyone that moves, I and many others believe that revs is in need of a rules rewrite and a mechanic rework so the gamemode can actually be enjoyable for all parties involved. It isn't fun when the rules prevent you from discerning what to do during revs or even how to handle expedited execution which got removed in the latest rule rewrite.

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**

🆑 
- remove: Removed revolutionary's pending a rules rewrite and mechanic rework.

### CODE REVIEW

**Comment by FATFSAAM2** [comment]
I agree

**Comment by slarticodefast** [comment]
This has been declined the last time the same PR was made #30758
Also you should add the 5% chance to the traitor gamemode so they still sum up to 100%.

**Comment by EdenTheLiznerd** [comment]
> Please add the 5% to survival instead

This is not a place to argue where the "5%" goes, I'd much rather you discuss what to actually do with revs as a gamemode.

**Comment by EdenTheLiznerd** [comment]
> This has been declined the last time the same PR was made #30758 Also you should add the 5% chance to the traitor gamemode so they still sum up to 100%.

Got it.

**Comment by LaOubliette** [comment]
I don’t think it should be outright removed, a lot of these are just plain player issues. Like, security just murdering people is as unreasonable against revs as it would be for any antag without a visual tell. 

Fixing revs would be as easy as adding a way for revheads to directly communicate with other revs or a visual tell, like custom armor they could build that would be worth outing themselves to wear.

**Comment by lzk228** [comment]
again?

**Comment by EdenTheLiznerd** [comment]
> again?

Please bring constructive criticism or actual ideas to my PR, I'd genuinely like this PR to go somewhere and have an actual effect on the revs discourse.

**Comment by LaOubliette** [comment]
> > I don’t think it should be outright removed, a lot of these are just plain player issues. Like, security just murdering people is as unreasonable against revs as it would be for any antag without a visual tell.
> 
> If a very large amount of different players (possibly unknowingly) break rules around a specific antagonist, chances are it's an issue with the antagonist, not with the players.

I mean i saw a security officer just say “fuck this” and kill a guy before waiting for mindshields, that is quite literally just a player issue.

**Comment by EdenTheLiznerd** [comment]
> > > I don’t think it should be outright removed, a lot of these are just plain player issues. Like, security just murdering people is as unreasonable against revs as it would be for any antag without a visual tell.
> > 
> > 
> > If a very large amount of different players (possibly unknowingly) break rules around a specific antagonist, chances are it's an issue with the antagonist, not with the players.
> 
> I mean i saw a security officer just say “fuck this” and kill a guy before waiting for mindshields, that is quite literally just a player issue.

Every gamemode has player issues, it just happens that revs as a gamemode is severely flawed and puts more strain on the players than anything.

**Comment by LaOubliette** [comment]
> > > > I don’t think it should be outright removed, a lot of these are just plain player issues. Like, security just murdering people is as unreasonable against revs as it would be for any antag without a visual tell.
> > > 
> > > 
> > > If a very large amount of different players (possibly unknowingly) break rules around a specific antagonist, chances are it's an issue with the antagonist, not with the players.
> > 
> > 
> > I mean i saw a security officer just say “fuck this” and kill a guy before waiting for mindshields, that is quite literally just a player issue.
> 
> Every gamemode has player issues, it just happens that revs as a gamemode is severely flawed and puts more strain on the players than anything.

Revs has always been a chaotic game mode but I don’t think removal is necessary when it could be so easily tweaked, especially when 14 has so few gamemodes.

Adding a way to make “rev comm key” or heck even forcing a visual tell ala bloodcult win the reach a level of heads dead or crew converted would go very far.

**Comment by Kadeo64** [comment]
image

this gamemode man.

**Comment by EdenTheLiznerd** [comment]
> Revs has always been a chaotic game mode but I don’t think removal is necessary when it could be so easily tweaked, especially when 14 has so few gamemodes.
> 
> Adding a way to make “rev comm key” or heck even forcing a visual tell ala bloodcult when they reach a level of heads dead or crew converted would go very far.

Revs isn't being "removed" it's being disabled so people don't need to deal with it while changes are implemented, revs in itself needs more mechanics and better rules, the gamemode is painful to experience as security and command if you actually plan on following the rules.

**Comment by EthanQix** [comment]
Revs Disabled 2 : Electric Boogaloo

Now let's see if the game mode actually gets fixed (or changed at all) before being reintroduced 👍

**Comment by Kadeo64** [comment]
> Revs Disabled 2 : Electric Boogaloo
> 
> Now let's see if the game mode actually gets fixed (or changed at all) before being reintroduced 👍

i'm all in on nothing happening with revs and it getting reintroduced with a single rule change 3 months in the future

**Comment by EdenTheLiznerd** [comment]
> > Revs Disabled 2 : Electric Boogaloo
> > Now let's see if the game mode actually gets fixed (or changed at all) before being reintroduced 👍
> 
> i'm all in on nothing happening with revs and it getting reintroduced with a single rule change 3 months in the future

Again, all I ask is that the topic of this PR remain strictly productive and remain without shitposts.

**Comment by EthanQix** [comment]
More seriously though, the main design issue with Revolutionaries game mode is that each team has an objective of "killing the opposing commanders" (with capturing added as an afterthought) so the round basically boils down to team death match. It also becomes very easy for said commanders to hide and stall the round.

In the same way Nukies work, Revs need a static target that can be defended rather than 5 targets that can hide forever.

**Comment by EthanQix** [comment]
> More seriously though, the main design issue with Revolutionaries game mode is that each team has an objective of "killing the opposing commanders" (with capturing added as an afterthought) so the round basically boils down to team death match. It also becomes very easy for said commanders to hide and stall the round.
> 
> In the same way Nukies work, Revs need a static target that command must defend rather than 5 targets that can hide forever.

Either that or mindshields need to double as tracking implants so command can't hide.

**Comment by Kadeo64** [comment]
> Again, all I ask is that the topic of this PR remain strictly productive and remain without shitposts.

Alright then, here's my non-joke take about revs.

It's unsalvageable. It requires roleplay on LRP to avoid turning into a huge murderbone fest (which we already have covered with zombies/nukies) and on MRP it requires a lot of anti-metagaming. You'd have to somehow incentivize taking unconvertables as hostages in order to prevent the mass executions. And the main win conditions being roughly 9 players that can go and fuck off around the station for an hour is incredibly annoying. 

I think that revolutionaries was a fine gamemode the first month or two when it was new and people weren't metagaming it as hard (i'm totally not biased because I was one of, if not the first revolutionaries converted in an actual round), but it just needs to be removed completely and refactored into something completely different. The novelty of fighting brainwashed crew with tider equipment wore off. Other conversion antags like cults (and /tg/ xenomorphs :trollface:) would be much more fun.

**Comment by Everturning** [comment]
plural of revolutionary is revolutionaries, not "revolutionary's" fyi.
i dont have any opinions on the change tho

**Comment by Kadeo64** [comment]
> plural of revolutionary is revolutionaries, not "revolutionary's" fyi. i dont have any opinions on the change tho

we got a grammar revolutionary over here

**Comment by Aisu9** [comment]
> I don’t think it should be outright removed, a lot of these are just plain player issues

players issues that admins struggle to deal with, additional reason to disable this antag

**Comment by Iamgoofball** [comment]
I feel like this is yet another case of SS14 deciding to reimplement something from SS13 from first principles without actually learning the lessons of SS13 because of a dogmatic insistence that you're not SS13 despite using both the visual style and core game design of SS13, with a few added party tricks and a nicer engine.

Mindshields should be removable because what's nearing on 20 years of Revolution as a gamemode at /tg/, /vg/, bay, goon, etc. has taught us that removable mindshields provides a way to flip people to the revolution side who otherwise wouldn't be achievable, so you should implement that

Kicking the heads of staff off of the station Z-level is another viable strategy for revolutionaries along with a timeout system for people who sit in a locker for 5 hours stops the gamemode pacing from getting fucked while providing an alternative to just flat murdering every head of staff(although murder is usually the faster option), which we learned via 20 years of Revolution as a gamemode at /tg/, /vg/, bay, goon, etc.

You don't need to add any additional tells, you don't need to overcomplicate it, just stick to the existing, tried and tested formula of Revolution from SS13 and quit trying to change something that works just fine. Revolution is a notoriously finicky gamemode to change without breaking it.  It's a very delicate balance. If all the pieces aren't in place just right from a design standpoint, it breaks. We've painstakingly found the correct pieces and their correct positions over the last 20 years of development at SS13.

Please, learn from that, and stop trying to re-learn how to make a wheel from first principles every 3 months.

**Comment by metalgearsloth** [comment]
> If all the pieces aren't in place just right from a design standpoint, it breaks. We've painstakingly found the correct pieces and their correct positions over the last 20 years of development at SS13.

Why didn't you do that for heretic.

**Comment by mirrorcult** [comment]
i dont think the revs gameplay experience between SS13 and SS14 is all that different (there arent that many changes at all, the actual mechanics are ported pretty much 1:1 barring edge case stuff). people mostly are complaining about it being a murderbone fest, which was absolutely also true of TG revs when i still played--i think this is more a player expectations issue than anything

**Comment by mirrorcult** [comment]
its just very strange to ascribe the problem to "idiot ss14 designers are overthinking and ruining things that were already perfect for no reason" when revs was ported with basically zero design scrutiny beyond "probably do what ss13 did"

**Comment by mirrorcult** [comment]
if anything, the issues arose from us porting an SS13 gamemode without properly thinking about how it would properly fit into SS14 holistically! sounds like we should have done it from first principles instead

**Comment by Golinth** [comment]
> Why didn't you do that for heretic.
Because they have only been around for 4 years, not 20 :godo:

**Comment by cohanna** [comment]
I do think there should be a way to remove mind shields, but I don't see revs as the "unfun muderbone garbage" it's often presented as in a lot of these discourses. 

The main issue is player culture/expectation not the game mode itself. The player culture around ss14 has resulted in every non traitor round being seen as a negative unless you are the antag for that round. No one likes war ops until they are the nukie. Revs unless they are the headrev. Zombies unless they are the initial infected. I have even seen this sentiment in response to admemes (as rare as those are). It's as if novelty is a negative.

Revs is already an extremely infrequent game mode (only 5% of rounds are rev rounds) and despite being weighted the same as zombies, zombies are seen more often due to its ability to appear late round in other game modes. So it's not as if revs are, in any realistic capacity, stopping people from playing command or sec roles. Furthermore, the process of identifying a revolution has been made drastically easier with the introduction of the AI. Giving both sec and command the ability to respond adequately via early warning. A big reason sec and command don't know how to handle revs in the first place is BECAUSE it's so infrequent.

I have also seen the recruitment method of revs be a point of contention since it often has no RP associated with it. "Why should I help you/join your side? I've never interacted with you until you walked up and flashed me." Is a legitimate statement I have seen. This feels like a fundamental misunderstanding of what the revs game mode is. Revs are essentially a mind controlled swarm with the head revs being the only "independent thinkers" which is surprisingly something many struggle with grasping. 

As we add more antags, and subsequently more gamemodes, the problem of "TDM" will continue to arise. Cults in particular will be a point of contention since, unlike revs, once the cult becomes big enough everyone is a hostile target (not just sec, command, and the mind shielded) and recruitment largely stops. If the player base continues to expect only easily perceivable and/or preventable threats 100% of the time wizards will never make it into this game either. 

Revs will never be accepted until the expectation of a SS14 round becomes more chaotic and this is an issue that removing revs only hurts.

**Comment by EthanQix** [comment]
> As we add more antags, and subsequently more gamemodes, the problem of "TDM" will continue to arise. Cults in particular will be a point of contention since, unlike revs, once the cult becomes big enough everyone is a hostile target (not just sec, command, and the mind shielded) and recruitment largely stops. If the player base continues to expect only easily perceivable and/or preventable threats 100% of the time wizards will never make it into this game either.

I agree on the fact that SS14 needs more gamemodes, but the main problem with Revs is RP and game mechanics being misaligned : 
- from a RP perspective, command and sec are supposed to protect the crew by finding and disabling the people mindcontrolling them, while the revolutionaries are trying to take over the station from NT.
- from a game mechanics perspective, command and sec need to kill the headrevs, and the revolutionaries need to kill/capture every command role.

Unfortunately when game mechanics and RP don't align, game mechanics tend to take over. In this case, the most straightforward way to "win" is to just kill the opposing faction. When command and sec kill the whole crew they are supposed to protect, this is described as a win for their side on the EoR summary. When revs destroy half the station they are supposed to take over but manage to kill command, this is described as a win for their side on the EoR summary. The game itself pushes the TDM mentality through positive reinforcement.

Long story short, the objectives for the rev gamemode need to be redesigned, so that mass murder/destruction isn't a viable strategy anymore.

**Comment by cohanna** [comment]
> Unfortunately when game mechanics and RP don't align, game mechanics tend to take over. In this case, the most straightforward way to "win" is to just kill the opposing faction. When command and sec kill the whole crew they are supposed to protect, this is described as a win for their side on the EoR summary. 

I don't fully agree that the RP and mechanics don't align. It's not like sec is going around round removing revs and, if they have the ability, would much rather mindshield over killing purely because it reduces rev numbers and bolsters their own. If you would like to take this further a simple change to the EoR summary (adding a crew %evaced for a command win) would further reinforce keeping crew alive, or at the very least reviving them.


> When revs destroy half the station they are supposed to take over but manage to kill command, this is described as a win for their side on the EoR summary.

Revs, by rules, shouldn't be destroying half the station and should keep it as intact as possible. While this can't be helped with a simple EoR summary change it CAN be helped with a single popup when revs get converted (same as when they get reconverted) saying their objective (kill/capture command, do so with minimal station damage.) hell there can even be an explicit line that says don't loose. 

These are all changes that can be made without a whole revs refactor that will improve player perception. 

I doubt a refactor would even occur due to how finicky the revs game mode is in the first place as almost any change to it will just make it into a reskinned cult.

it feels as if the ss14 community as a whole are two quick to go "I don't like this remove it" instead of attempting any changes to fix it, dismissing them all as bandaid changes. Incremental bandaid changes are what's needed, it's what crafted ss13 and it's what will craft 14. We can just do it faster since we can envision what a final product looks like.

**Comment by SpeltIncorrectyl** [comment]
Here before it happens a third time.

**Comment by UltimateJester** [comment]
I have no reason to see why this is till open as of the moment.
